src/core/Table/columns/selectionColumn.tsx

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { CellProps, CellRendererProps, HeaderProps } from 'react-table';
+import { Checkbox } from '../../Checkbox';
+import { DefaultCell } from '../cells';
+
+export const SELECTION_CELL_ID = 'iui-table-checkbox-selector';
+
+export const SelectionColumn = <T extends Record<string, unknown>>(
```

Comment:
Maybe add a JSDoc comment above this because it is exposed to the user.

Same thing in ExpanderColumn and ActionColumn.

Reply:
Something like this? https://github.com/iTwin/iTwinUI-react/pull/599/commits/aa3f0d97909336142ecc5920e2b4fb97f6639d07

Reply:
Yeah that's better. I think `SelectionColumn` and `ExpanderColumn` could also use `@example` because they accept parameters.

Also, you're going to hate me for saying this but, now I'm realizing that we could probably align the parameters so that these columns accept an object (similar to `ActionColumn`),

e.g. usage could go from this:
```ts
SelectionColumn(() => true)
```

to this:
```ts
SelectionColumn({ isDisabled: () => true })
```

Reply:
Good that you raised this question as I forgot to ask everyone's opinion whether to have regular arguments or props object. In order to align with ActionColumn, I changed to props object.